pagination/pagetoken_test.go

```diff
@@ -34,7 +35,37 @@ func TestParseOffsetPageToken(t *testing.T) {
 		assert.NilError(t, err)
 		assert.Equal(t, int64(30), pageToken3.Offset)
 	})
-
+	t.Run("skip", func(t *testing.T) {
+		t.Run("handle empty token with skip", func(t *testing.T) {
+			request1 := &freightv1.ListSitesRequest{
+				Parent: "shippers/1",
+				Skip:   30,
+			}
```

Comment:
It's a little bit weird to not provide a `PageSize` argument here (even though it won't affect the test); what does it mean to List 0 resources? [AIP-158](https://google.aip.dev/158) say that if the client doesn't provide a `page_size`, the "API chooses an appropriate default".